python/src/trezorlib/stellar.py

```diff
@@ -254,9 +254,9 @@ def _parse_operation_bytes(unpacker):
         )
 
         if op.asset_type == ASSET_TYPE_ALPHA4:
-            op.asset_code = unpacker.unpack_fstring(4)
+            op.asset_code = unpacker.unpack_fstring(4).decode("utf-8")
```

Comment:
"UTF-8" is the default value so no need to specify it, could you remove the string?